lib/internal/Magento/Framework/App/Response/Http.php

```diff
@@ -131,7 +133,7 @@ public function sendVary()
      */
     public function setPublicHeaders($ttl)
     {
-        if ($ttl < 0 || !preg_match('/^[0-9]+$/', $ttl)) {
+        if ($ttl < 0 || !is_numeric($ttl)) {
```

Comment:
We are not looking for code changes with these issues, please revert as it was.

Reply:
This change introduce since i added strict_type at the first line of fine!

Reply:
In this case, let's create a separate PR to add the strict type and the code change. On this scope, we are looking just for API marking.

Reply:
updating it